src/constants/index.js

```diff
@@ -0,0 +1,3 @@
+const SIGNUP = 'SIGNUP';
```

Comment:
Let's have the contents of this file as:
```
const CONSTANTS = {
   SIGNUP,
   LOGIN,
   ....
};

export default CONSTANTS;
```

So all our constants will be placed within this object. Then when we want to access a particular constant, what we will do is:

```
import constants from '../_constants';

const { SIGNUP, ... } = constants;
```